src/main/java/com/vinylteam/vinyl/web/servlets/SignUpServlet.java

```diff
@@ -49,24 +49,30 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String discogsUserName = request.getParameter("discogsUserName");
         attributes.put("email", email);
         attributes.put("discogsUserName", discogsUserName);
-        if (!password.equals(confirmPassword)) {
+        if (password.equals("")){
```

Comment:
Add same checks to front on profile as you have on /signUp: password at least 8 characters, uppercase, lowercase, number.

Reply:
we have checks in editProfile and registration. I think that it's enough